<launch>
			 <node pkg = "radar_omnipresense" type= "radar_publisher" name= "radar_1_data_and_services"/>
			        <remap from="radar" to="topic1"/>
			        <param name="serialPort" value="/dev/ttyACM0"/>
 </launch>
